bin/scan

```diff
@@ -5,30 +5,6 @@ ignored=(
     # Description: Escape tab switching cookie name in browsable API.
     # Reason: We do not use the browsable API
     25804
-    # django, installed 1.6.11, affected <1.7.11, id 25714
-    # Description: The get_format function in utils/formats.py in Django before 1.7.x before 1.7.11, 1.8.x before 1.8.7, and 1.9.x before 1.9rc2 might allow remote attackers to obtain sensitive application secrets via a settings key in place of a date/time format setting, as demonstrated by SECRET_KEY.
-    # Reason: We don't leverage this feature.
-    25714
-    # django, installed 1.6.11, affected <1.7.6, id 25715
-    # Description: Cross-site scripting (XSS) vulnerability in the contents function in admin/helpers.py in Django before 1.7.6 and 1.8 before 1.8b2 allows remote attackers to inject arbitrary web script or HTML via a model attribute in ModelAdmin.readonly_fields, as demonstrated by a @property.
-    # Reason: We don't leverage this feature.
-    25715
-    # django, installed 1.6.11, affected <1.8.10, id 33073
-    # Description: The utils.http.is_safe_url function in Django before 1.8.10 and 1.9.x before 1.9.3 allows remote attackers to redirect users to arbitrary web sites and conduct phishing attacks or possibly conduct cross-site scripting (XSS) attacks via a URL containing basic authentication, as demonstrated by http://mysite.example.com\@attacker.com.
-    # Reason: We have already patched around this and use our own `is_safe_url` function pulled from Django 1.10.
```

Comment:
I assume we removed the monkey patch already?

Reply:
Oh, no, I doubt it. But it doesn't particularly for this context. Probably just good hygiene anyways.

Reply:
No. I'll track that down and see if it can be excised.

Reply:
I took a look and the `is_safe_url` we're using comes from Matt's [django-sudo](https://github.com/mattrobenolt/django-sudo/blob/master/sudo/utils.py) which we're using for a few other things. There isn't anything to remove from sentry at this point in time.